.github/buildomat/jobs/phd-run.sh

```diff
@@ -55,7 +55,7 @@ ls $propolis
 # Disable errexit so that we still upload logs on failure
 set +e
 
-(RUST_BACKTRACE=1 ptime -m pfexec $runner \
+(RUST_BACKTRACE=1 RUST_LOG=debug ptime -m pfexec $runner \
```

Comment:
Optional: I found myself using `RUST_LOG=info,phd_framework=debug` while working on my most recent changes, just to keep other RUST_LOG respecters (I think hyper is one of them?) out of the logs. But I don't feel especially strongly here.

Reply:
ah, whoops, I meant to do `RUST_LOG=info,phd=debug` but somehow forgot to commit it! will fix!